modules/apps/commerce/commerce-shipping-engine-fixed-service/src/main/java/com/liferay/commerce/shipping/engine/fixed/internal/upgrade/CommerceShippingEngineFixedUpgradeStepRegistrator.java

```diff
@@ -70,6 +71,9 @@ public void register(Registry registry) {
 		registry.register(
 			"2.2.0", "2.3.0", new CommerceShippingFixedOptionUpgradeProcess());
 
+		registry.register(
+			"2.3.0", "2.4.0", AccountEntryShippingOptionRelTable.create());
```

Comment:
@CrescenzoRegaLF you're missing the bnd.bnd version change

Reply:
I added it in the commit
COMMERCE-8731 regenerate the service